### PR TITLE
Add more checks

### PR DESCRIPTION
**Pull request by dz0ny**
[Add secure entry into terminal checks](https://github.com/ParetoSecurity/pareto-mac/commit/2fc0cc1e14f91c5552d5578117c64cae8958cba6)

[Add App store updates check](https://github.com/ParetoSecurity/pareto-mac/commit/88dff7ba27f9eb4bb453fba08c339d07c5d48b9d)

### CODE REVIEW

File: Pareto/Checks/System Integrity/SecureTerminal.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    override var UUID: String {
+        "5cbe1cfd-ff28-4cc7-8998-5d72e608b28d"
+    }
+    
+    override var TitleON: String {
+        "Other apps cannot read from Terminal.app"
+    }
+    
+    override var TitleOFF: String {
+        "Other apps can read from Terminal.app"
```
Comment (zupo) [inline Pareto/Checks/System Integrity/SecureTerminal.swift:24]: "Other apps can detect and record what you type in Terminal."

File: Pareto/Checks/System Integrity/SecureiTerm.swift [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import Version
+
+class SecureiTermCheck: ParetoCheck {
+    static let sharedInstance = SecureiTermCheck()
+    override var UUID: String {
+        "6cbe1cfd-ff28-4cc7-8998-5d72e608b28d"
+    }
+    
+    override var TitleON: String {
+        "Other apps cannot read from iTerm.app"
```
Comment (zupo) [inline Pareto/Checks/System Integrity/SecureiTerm.swift:20]: same as above

File: Pareto/Checks/System Integrity/SecureiTerm.swift [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    }
+    
+    override var TitleOFF: String {
+        "Other apps can read from iTerm.app"
```
Comment (zupo) [inline Pareto/Checks/System Integrity/SecureiTerm.swift:24]: same as above

**Review by zupo — APPROVED** [review]

File: Pareto/Checks/System Integrity/SecureTerminal.swift (hunk 1 above)
Comment (dz0ny) [reply Pareto/Checks/System Integrity/SecureTerminal.swift:24]: image

long titles are problematic, same goes for menubar app

Comment (zupo) [reply Pareto/Checks/System Integrity/SecureTerminal.swift]: Does this fit?

"Other apps can't record typing in Terminal"?

**Pull request merged**